pkgs/development/ocaml-modules/bisect_ppx-ocamlbuild/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, fetchFromGitHub, ocaml, dune, findlib, ocamlbuild }:
+
+let version = "1.4.0"; in
+
+stdenv.mkDerivation {
```

Comment:
We have now a `buildDunePackage` since https://github.com/NixOS/nixpkgs/pull/49684